edweiss-app/model/quizzes.ts

```diff
@@ -31,13 +31,15 @@ namespace Quizzes {
 
     export type Exercise = MCQ | TF;
 
-    export interface Quiz {
-        name: string;
-        deadline?: string;
+    export interface Quiz extends Assignment {
         exercises: Exercise[];
         answers: QuizzesAttempts.Answer[];
         ended: boolean;
         showResultToStudents: boolean;
+        // inherits:
+        // name: string
+        // type: AssignmentType = "quiz"|"submit"
+        // dueDate: TimeStamp
```

Comment:
You can delete this comments

Reply:
This was just to clarify what attributes Quiz inherits. If you think this is redundant, I can delete.